services/QuillLMS/db/migrate/20210421181107_create_feedback_history_flags.rb

```diff
@@ -0,0 +1,9 @@
+class CreateFeedbackHistoryFlags < ActiveRecord::Migration
+  def change
+    create_table :feedback_history_flags do |t|
+      t.references :feedback_history, null: false
+      t.string :flag, null: false
```

Comment:
how about making this jsonb for extensibility?

Reply:
I'm not convinced that this has immediate value, which leads me to think we should hold off on doing this unless/until we see a reason to.  Also, some of the query code currently relies on this being a string, and while it wouldn't be that hard to modify, it doesn't seem like there's a good reason to yet.